source/OctoVersion.Tests/ConfigurationBootstrapperFixture.cs

```diff
@@ -15,12 +15,15 @@ public class ConfigurationBootstrapperFixture : IDisposable
     {
         public void Dispose()
         {
-            foreach (DictionaryEntry environmentVariable in Environment.GetEnvironmentVariables())
-            {
-                var key = (string)environmentVariable.Key;
-                if (key.StartsWith(ConfigurationBootstrapper.EnvironmentVariablePrefix))
-                    Environment.SetEnvironmentVariable(key, "");
-            }
+            var envVars = Environment.GetEnvironmentVariables()
+                .Cast<object?>()
+                .Where(environmentVariable => environmentVariable != null)
+                .Select(environmentVariable => (DictionaryEntry)environmentVariable!)
+                .Select(envVar => (string)envVar.Key)
+                .Where(key => key.StartsWith(ConfigurationBootstrapper.EnvironmentVariablePrefix));
+
+            foreach (var key in envVars)
+                Environment.SetEnvironmentVariable(key, "");
```

Comment:
I can see why this is here, but it really feels like this should actually handle setting/clearing env vars in a more robust way. Or we should think about better ways to test this configuration source.

Reply:
I agree this isn't great, but also, I cant think of a better way.
Thinking of the opportunity cost, I think this is okay to go 🙈 on.